spec/implementations/pto/fetch_pto_from_drive_spec.rb

```
@@ -0,0 +1,63 @@
+require 'bas/shared_storage/postgres'
+require 'bas/shared_storage/default'
+require 'date'
+require_relative '../../../src/implementations/fetch_pto_from_drive'
+
+RSpec.describe Implementation::FetchPtosFromGoogleSheets do
+  let(:mocked_shared_storage_writer) { instance_double(Bas::SharedStorage::Postgres) }
+  let(:mocked_shared_storage_reader) { instance_double(Bas::SharedStorage::Default) }
+
+  before do
+    options = {
+        spreadsheet_id: 'FAKE_SPREADSHEET_ID',
```

RuboCop (1.75.5)

[convention] Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.

(Layout/FirstHashElementIndentation)